packages/server/generate-schema.js

```diff
@@ -0,0 +1,52 @@
+const { spawn } = require('effection');
+const { main, exec } = require('@effection/node');
+
+main(function*() {
+  // get the path to the `ts-node` binstub.
+  // `@effection/node#exec() uses the shellwords package to split
+  // command and arguments, but that has a bug that causes it to
+  // to treat `\` as a literal indicator and not a path separator
+  // which causes it to fail on windows, so we have to do a bit of
+  // munging to a unix path separator, which is then (thankfully)
+  // re-munged by cross-spawn.
+  // See https://github.com/jimmycuadra/shellwords/issues/10
+  let tsNode = (yield getsh('yarn bin ts-node'))
+      .trim().replace(/\\/g, '/');
+
+  yield sh(`${tsNode} ./src/schema.ts`, {
+    env: {
+      PATH: process.env.PATH,
+      BIGTEST_GENERATE_SCHEMA: 'true'
+    }
+  });
+});
+
+function* sh(command, options) {
+  console.log(`$ ${command}`);
+  let child = yield exec(command, options);
+
+  yield spawn(child.stdout.subscribe().forEach(function*(datum) {
+    process.stdout.write(datum);
+  }));
+
+  yield spawn(child.stderr.subscribe().forEach(function*(datum) {
+    process.stderr.write(datum);
+  }));
```

Comment:
Can't we just use `stdio: inherit`?

Reply:
We actually don't export the `stdio` option in your `exec()` api https://github.com/thefrontside/effection/blob/v0/packages/node/src/exec/api.ts#L25-L47

We could add that to `exec()`, but now that I think of it, that was always a pretty confusing API. Maybe we should add `sh()` to `@effection/node`?

Reply:
If we "inherit", then we can't act on any of the output, essentially a pure side effect. Using "inherit" is what nearly every use of child_process does which is partly why we are likely one of the few that has run into the deep issues on windows that we have had. Maybe another function makes sense, but I wonder if the use case might be outside the scope of this package?

Reply:
Yeah, it's a dang useful function, but forcing the the user to decide where the bytes go feels like the right primitive.